src/commands/plugins.rs

```diff
@@ -39,6 +39,9 @@ pub enum PluginCommands {
 
     /// Fetch the latest Spin plugins from the spin-plugins repository.
     Update,
+
+    /// Print information about a plugin.
+    Inspect(Inspect),
```

Comment:
`inspect` feels like it oversells this command - I always assume I'm going to get detailed, even internal, information about a thing when I inspect it.  How would you feel about `info` or `show` instead?  Too vague?

Reply:
I'm cool with renaming it to `info` or `show`. I was just using same vocabulary as `brew` does.

Will change it to `show`